modules/protobuf/src/main/scala/smithy4s/protobuf/Protobuf.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ *  Copyright 2021-2024 Disney Streaming
+ *
+ *  Licensed under the Tomorrow Open Source Technology License, Version 1.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     https://disneystreaming.github.io/TOST-1.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package smithy4s.protobuf
+
+import smithy4s.schema._
+
+object Protobuf {
+
+  def codecs: CachedSchemaCompiler[ProtobufCodec] = ProtobufCodec
```

Comment:
**question:** shouldn't this be a val? It seems to be the case for xml/json.

Reply:
Not necessarily, because `ProtobufCodec` itself is a singleton and this is just forwarding the reference.

Reply:
hmmmmm fair nuf